kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -198,10 +202,29 @@
     },
     $trNameSpace: 'userPage',
     $trs: {
-      noUsersExist: 'No Users Exist.',
-      allUsersFilteredOut: 'No users match the filter.',
+      // input & accessibility labels
```

Comment:
There is no local namespacing. I think camelcasing is probably sufficient, unless we are getting hugely complex things going on.